spec/graphql/rake_task_spec.rb

```diff
@@ -31,7 +31,7 @@ def self.visible?(member, ctx)
 # Configured task
 GraphQL::RakeTask.new(idl_outfile: "tmp/configured_schema.graphql") do |t|
   t.namespace = "graphql_custom"
-  t.load_context = ->(task) { {filtered: true} }
+  t.load_context = ->(task) { {filtered: true, skip_types_migration_error: true } }
```

Comment:
TODO: this was necessary because `all_types` included introspection in one case, but not the other. It didn't end up mattering, since it was in the context of printing the schema. (Maybe that's the link -- the other weird issue was in `printer_spec.rb`.) 

But I need to figure out why those things were on the list in one case but not the other.

Reply:
OK, I got it: 

The previous implementation called `Warden#reachable_types` in the printer, see the diff here: https://github.com/rmosolgo/graphql-ruby/pull/4998/files#diff-e9dd0d295a58b66fabd1cf717040de5a78ade0feac6aeabae5247b5785c029abL268

But `reachable_types` never actually called `.visible?` on root types, it adds types immediately: 

https://github.com/rmosolgo/graphql-ruby/blob/83dabed599639c662b8d209d4218c1ad7cc1f1ab/lib/graphql/schema/warden.rb#L509-L510

So, introspection types were not checked through `.visible?` in that case. (Neither were root types or orphan_types -- I think they should have been.) 

But `Subset` passes them all through `.visible?` before adding them to this list. So I think Warden has the wrong behavior here. I'm going to update the `.visible?` implementation to permit introspection stuff.